srcpkgs/buildah/template

```diff
@@ -14,15 +14,16 @@ short_desc="Dockerfile compatible OCI image building tool"
 maintainer="Cameron Nemo <cnemo@tutanota.com>"
 license="Apache-2.0"
 homepage="https://github.com/containers/buildah"
+changelog="https://github.com/containers/buildah/blob/master/CHANGELOG.md"
 distfiles="${homepage}/archive/v${version}.tar.gz"
-checksum=355b7b6c6486be0375d6ac74c43a61f1bed1faa6c90b1f70c389da4a7fc9a03e
+checksum=500c82fb369fd08a9bb9a41f294c5975f17575770dbfaba5cce9a6e09b6a3fb9
 
 post_build() {
 	make -C docs GOMD2MAN=go-md2man
 }
 
 post_install() {
-	vinstall contrib/completions/bash/buildah 644 /usr/share/bash-completion/completions
+	vcompletion contrib/completions/bash/buildah bash buildah
```

Comment:
You can omit `buildah`, it will use the package's name by default.

```suggestion
	vcompletion contrib/completions/bash/buildah bash
```

Reply:
Done.